python_ta/typecheck/base.py

```diff
@@ -268,6 +268,10 @@ def can_unify(self, t1, t2):
                 return True
             else:
                 return False
+        elif isinstance(t1, GenericMeta):
```

Comment:
I think I already commented on this change in a previous pull request. You should be able to remove this.